### ISSUE TITLE
Handle PDFSyntaxError in convert_pdf_to_text

### ISSUE THREAD

**Issue by mchamberland** [cutright/IMRT-QA-Data-Miner#12]
We should try to skip bad PDF files to prevent a single bad file stopping the entire analysis:

> Traceback (most recent call last):
>   File "c:\Users\m310767\.vscode\extensions\ms-python.python-2019.11.50794\pythonFiles\ptvsd_launcher.py", line 43, in <module>
>     main(ptvsdArgs)
>   File "c:\Users\m310767\.vscode\extensions\ms-python.python-2019.11.50794\pythonFiles\lib\python\old_ptvsd\ptvsd\__main__.py", line 432, in main
>     run()
>   File "c:\Users\m310767\.vscode\extensions\ms-python.python-2019.11.50794\pythonFiles\lib\python\old_ptvsd\ptvsd\__main__.py", line 342, in run_module
>     run_module_as_main(target, alter_argv=True)
>   File "c:\Users\m310767\AppData\Local\Continuum\anaconda3\envs\py3\lib\runpy.py", line 193, in _run_module_as_main
>     "__main__", mod_spec)
>   File "c:\Users\m310767\AppData\Local\Continuum\anaconda3\envs\py3\lib\runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "C:\Users\m310767\dev\IMRT-QA-Data-Miner\IQDM\main.py", line 150, in <module>
>     main()
>   File "C:\Users\m310767\dev\IMRT-QA-Data-Miner\IQDM\main.py", line 140, in main
>     no_recursive_search=args.no_recursive_search)
>   File "C:\Users\m310767\dev\IMRT-QA-Data-Miner\IQDM\main.py", line 63, in process_files
>     process_file(file_path, output_file, output_dir)
>   File "C:\Users\m310767\dev\IMRT-QA-Data-Miner\IQDM\main.py", line 68, in process_file
>     row, report_type, columns = pdf_to_qa_result(file_path)  # process file
>   File "C:\Users\m310767\dev\IMRT-QA-Data-Miner\IQDM\main.py", line 30, in pdf_to_qa_result
>     text = convert_pdf_to_txt(abs_file_path)
>   File "C:\Users\m310767\dev\IMRT-QA-Data-Miner\IQDM\pdf_to_text.py", line 31, in convert_pdf_to_txt
>     check_extractable=True):
>   File "c:\Users\m310767\AppData\Local\Continuum\anaconda3\envs\py3\lib\site-packages\pdfminer\pdfpage.py", line 129, in get_pages
>     doc = PDFDocument(parser, password=password, caching=caching)
>   File "c:\Users\m310767\AppData\Local\Continuum\anaconda3\envs\py3\lib\site-packages\pdfminer\pdfdocument.py", line 585, in __init__
>     raise PDFSyntaxError('No /Root object! - Is this really a PDF?')
> pdfminer.pdfparser.PDFSyntaxError: No /Root object! - Is this really a PDF?

In this case, the offending file is a broken PDF of 0 size.

**Comment by cutright** [comment]
I used to have a generic try/except clause, but mistakenly thought I found possible crashes.  Currently process_file looks like this:

https://github.com/cutright/IMRT-QA-Data-Miner/blob/0d7147d7f7914ef47d636c8df970d3615a6eb207/IQDM/main.py#L66-L80

I'd prose the following instead (in the issue_12 branch):
https://github.com/cutright/IMRT-QA-Data-Miner/blob/02db4b9cafdbfa18866142f5f1b9e7f9ade422dd/IQDM/main.py#L66-L83

I don't recall why I restricted the original exception to TypeError, but it seems like the only place this could crash is in `pdf_to_qa_result()`.  I'll give this a shot tomorrow.
